feeds.xsh

```diff
@@ -0,0 +1,24 @@
+import asyncio
+import threading
+import time
+from feedy import Feedy
+
+app = Feedy('feeds.shelve')
+
+@app.add('https://stackoverflow.com/feeds/tag?tagnames=xonsh&sort=newest')
+def stackoverflow_question(info, body):
+    # site_title, site_subtitle, site_url, fetched_at, article_title, article_url, published_at, updated_at
+    msg = "New question on StackOverflow: {article_title} {article_url}".format(**info)
+    gitter_send_msg(msg)
+    send_irc_msg(msg)
+
+
+def run():
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+    while True:
+        app.run()
+        time.sleep(60*60*60)  # 1 hour
```

Comment:
Sounds fine.  Do you want to go ahead and make that change?  Then I'm happy to merge in and update the instance that's running in the rooms now.